sql/template_elasticdl.go

```diff
@@ -13,6 +13,45 @@
 
 package sql
 
+const elasticdlDataConversionTemplateText = `
+import os
+
+from elasticdl.python.common.constants import ODPSConfig
+from elasticdl.python.common.odps_io import ODPSReader
+from elasticdl.python.common.odps_recordio_conversion_utils import (
+    write_recordio_shards_from_iterator,
+)
+
+
+COLUMN_NAMES = {{.FeaturesList}}
+
+reader = ODPSReader(
+    os.environ[ODPSConfig.PROJECT_NAME],
```

Comment:
We can get the odps configuration from `datasource` argument like: 
https://github.com/sql-machine-learning/sqlflow/blob/113606eb853d6cd3a4d683d5bb5703620a6365e7/sql/codegen_alps.go#L175